apps/devhub/tasks.py

```diff
@@ -32,65 +32,116 @@
 log = logging.getLogger('z.devhub.task')
 
 
+# Simple wrapper for the sake of tests.
+def validate(file_, listed=None):
```

Comment:
Yes. The `listed` parameter only applies to `FileUpload` objects, so I want to be able to assert that it isn't being passed for `File` objects, which have a defined listed status.